src/transformers/modeling_electra.py

```diff
@@ -858,3 +858,133 @@ def forward(
             outputs = (total_loss,) + outputs
 
         return outputs  # (loss), start_logits, end_logits, (hidden_states), (attentions)
+
+
+class ElectraPooler(nn.Module):
```

Comment:
Instead of introducing a new class, I'd rather use the existing `SequenceSummary` which can do the pool + dropout (see the [`XLNetForMultipleChoice`](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_xlnet.py#L1369) for an example. It will require to add a few things to the config to make it work (see `XLNetConfig` for an example).

Reply:
>I think there is a way to make it more consistent with the other models for multiple choice and avoid introducing a new class, would you mind looking into it?

Sure, I wasn't aware of `SequenceSummary`.  I'll try to use it instead of `Pooler`

Reply:
But couldn't you just use the `ElectraClassificationHead` for that pooling 🤔:

https://github.com/huggingface/transformers/blob/86578bb04c9b34f9d8e35cd4fad42a85910dd9e9/src/transformers/modeling_electra.py#L346-L362

Reply:
Sadly no, because it adds a linear layer to `num_labels`